packages/gitbook/src/components/Adaptive/AINextPageSuggestions.tsx

```diff
@@ -0,0 +1,138 @@
+'use client';
+import { tcls } from '@/lib/tailwind';
+import { Icon, type IconName } from '@gitbook/icons';
+import { AnimatePresence, motion } from 'framer-motion';
+import Link from 'next/link';
+import { useEffect, useState } from 'react';
+import { useVisitedPages } from '../Insights';
+import { usePageContext } from '../PageContext';
+import { Emoji } from '../primitives';
+import { type SuggestedPage, useAdaptiveContext } from './AdaptiveContext';
+import { streamNextPageSuggestions } from './server-actions/streamNextPageSuggestions';
```

Comment:
Lets export it from `server-actions` so we don't have to do nested imports like this